build.sbt

```
@@ -14,12 +14,14 @@ lazy val microservice = Project("central-reference-data-inbound-orchestrator", f
     scalacOptions ++= Seq("-Wconf:src=routes/.*:s", "-Wconf:msg=Flag.*repeatedly:s"),
     resolvers += Resolver.jcenterRepo,
     PlayKeys.playDefaultPort := 7250,
+    Test / classLoaderLayeringStrategy := ClassLoaderLayeringStrategy.Flat
   )
 
 lazy val it = project
   .enablePlugins(PlayScala)
   .dependsOn(microservice % "test->test")
   .settings(DefaultBuildSettings.itSettings())
   .settings(libraryDependencies ++= AppDependencies.it)
+  .settings(Test/ classLoaderLayeringStrategy := ClassLoaderLayeringStrategy.Flat)
```

It's possible the build may complain about a missing space here
```suggestion
  .settings(Test / classLoaderLayeringStrategy := ClassLoaderLayeringStrategy.Flat)
```